src/tiny_slam/tiny_scan_cost_estimator.h

```diff
@@ -3,11 +3,29 @@
 
 #include "../core/grid_scan_matcher.h"
 
+/*!
+ * \brief Derived class from ScanCostEstimator to calculate "cost" of one scan
+ *
+ * This class presents one method to calculate a numeric value of one frame. This method is defined in the article.
+ */
 class TinyScanCostEstimator : public ScanCostEstimator {
 public:
+
+  /*!
+   * Function to estimate cost of values from laser scan which are given from every direction from one pose of robot.
```

Comment:
Consider: "Calculate a discrepancy between a given scan and a map assuming a given robot position using the approach described in the tinySLAM paper"